nlp_primitives/tests/test_lsa.py

```diff
@@ -109,3 +109,17 @@ def test_bad_algorithm_input_value(self):
         err_message = "TruncatedSVD algorithm must be either 'randomized' or 'arpack'"
         with pytest.raises(ValueError, match=err_message):
             LSA(algorithm="bad_algo")
+
+    def test_args_strings(self):
```

Comment:
can we add another scenario, where random_seed, corpus, and algorithm are all specified

Reply:
Added! [bd48d1d](https://github.com/alteryx/nlp_primitives/pull/161/commits/bd48d1d3eaa6cf17327bcd88242ba33e8b3fb54f)